Cython/Compiler/ExprNodes.py

```diff
@@ -14193,6 +14194,9 @@ def generate_result_code(self, code):
             checks = ["(%s != Py_None)" % self.arg.py_result()] if self.arg.may_be_none() else []
             checks.append("(%s(%s) != 0)" % (test_func, self.arg.py_result()))
             code.putln("%s = %s;" % (self.result(), '&&'.join(checks)))
+            code.putln("#if !CYTHON_ASSUME_SAFE_MACROS")
+            code.putln(code.error_goto_if_neg(self.result(), self.pos))
```

Comment:
I can rewrite it as

```
if ((!CYTHON_ASSUME_SAFE_MACROS) && %result% < 0) goto error
```

so the label is never unused. That might be slightly better (provided we manage to avoid "condition always true" warnings)